js/google_ima.js

```diff
@@ -147,6 +147,7 @@ require("../html5-common/js/utils/utils.js");
         this.adPlaybackStarted = false;
         this.vcPlayRequested = false;
         this.savedVolume = -1;
+        this.useGoogleAdUI = false;
```

Comment:
For the moment, yes.  If you don't have the ui enabled, it will do a clickthrough if you click on the video.  We may change the default later but since this is a very last minute change, before the cut, we are playing it safe and keeping the behavior we have at the moment.